src/Compiler/Compiler.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <IsShipped>true</IsShipped>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
this project doesn't have any output, so what's the value of it? It's purely something to help package

Reply:
It's a good question, I think I faced an issue [Which I can't recall from few months back] that I needed to ensure all packages are referenced to `.net8.0`

Can we merge this ticket and I will address them in a new PR? I try to avoid rebasing all of these commits